apps/aetestframework/lib/worker.ex

```diff
@@ -224,6 +224,19 @@ defmodule Aetestframework.Worker do
     send_command(node_name, "Oracle.respond(query_id, \"boolean\", 5)")
   end
 
+  def get_latest_tx_type(node_name) do
+    send_command(node_name, "%Block{txs: txs} = Chain.top_block")
+    send_command(node_name, "[tx] = txs")
+    send_command(node_name, "encoded_type = :erlang.term_to_binary(tx.data.type)")
```

Comment:
why is term_to_binary and then Base.encode32 used?

Reply:
Because the result that is received from the node is string and Base.encode32 is needed because it encodes term_to_binary result to the string in order to send it back to the node

Reply:
thy